apps/builder/app/builder/features/style-panel/sections/filter/filter-content.tsx

```diff
@@ -7,12 +7,9 @@ import {
   Flex,
```

Comment:
btw need to use code editor in all of those code areas instad of textarea, can be a separate pr though